app/models/assignment.rb

```diff
@@ -697,17 +691,6 @@ def next_criterion_position
     get_criteria.count > 0 ? get_criteria.last.position + 1 : 1
   end
 
-  # Returns the class of the criteria that belong to this assignment.
-  def criterion_class
```

Comment:
@david-yz-liu We do not need this class anymore. However, there is one use in automated_tests_helper in line 93 that I would like to delete. I was thinking that you may have a better understanding of why this line was added. I did see  that an index was created to have a proper association and faster queries, but I would like to hear your thoughts before I try to delete the above line tomorrow.